Jenkinsfile

```diff
@@ -58,50 +58,50 @@ pipeline {
       }
     }
 
-    stage('PyTorch Lightning DDP Checks') {
-      steps {
-        sh 'python "tests/core_ptl/check_for_ranks.py"'
-      }
-    }
-
-    stage('L0: Unit Tests GPU') {
-      steps {
-        sh 'pytest -m "not pleasefixme" --with_downloads'
-      }
-    }
-
-    stage('L0: Unit Tests CPU') {
-      when {
-        anyOf {
-          branch 'main'
-          changeRequest target: 'main'
-        }
-      }
-      steps {
-        sh 'CUDA_VISIBLE_DEVICES="" pytest -m "not pleasefixme" --cpu --with_downloads'
-      }
-    }
-
-    stage('L0: Computer Vision Integration') {
-      when {
-        anyOf {
-          branch 'main'
-          changeRequest target: 'main'
-        }
-      }
-      failFast true
-      parallel {
-        stage ('MNIST image classification with LeNet-5 Integration Test - on CPU') {
-          steps {
-            sh 'cd examples/cv && \
-            python mnist_lenet5_image_classification_pure_lightning.py trainer.gpus=0 \
-            trainer.accelerator=null \
-            trainer.fast_dev_run=true model.dataset.data_folder=/home/TestData \
-            && rm -rf outputs'
-          }
-        }
-      }
-    }
+//     stage('PyTorch Lightning DDP Checks') {
+//       steps {
```

Comment:
this should not be commented out?

Reply:
this was for debugging the reinstall issue, reverted